spec/support/matchers.rb

```diff
@@ -0,0 +1 @@
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
Exactly.  A negative matcher cannot be combined with any other matcher using `&`, so you can't say "this block changes the value of X this way but does not change the value of Y" in one sentence, requiring you to nest blocks.  I recently found this matcher quite useful and started to use it everywhere. 😁